python/Makefile

```diff
@@ -2,6 +2,7 @@
 
 VENV := venv
 MATURIN_VERSION := $(shell awk -F '[ ="]+' '$$1 == "requires" { print $$4 }' pyproject.toml)
+TARGET_WHL := $(wildcard ../target/wheels/*.whl)
```

Comment:
i wonder if this will result in any issue if there are multiple wheel files with different versions?

Reply:
I had the same thought, if there are any suggestions for picking the latest version or its on the dev to only have one version in the target/wheels directory.